src/display/api.js

```diff
@@ -1742,7 +1742,8 @@ var WorkerTransport = (function WorkerTransportClosure() {
             var size = width * height;
             var rgbaLength = size * 4;
             var buf = new Uint8Array(size * components);
-            var tmpCanvas = createScratchCanvas(width, height);
+            var canvasFactory = new DOMCanvasFactory();
```

Comment:
I think we need to keep this part without change as well and add `assert(typeof document !== 'undefined')`.

I'm thinking the canvasFactory must be a property of render parameters. To protect core from calling browser decoding (which will fail anyway at setting image src), we need to pass a flag to evaluator that shows that browser decoding is disabled. I thought we do/did it, since pdf2svg example worked, but I cannot find this code now, so we need to pass this flag with getOperatorList requests.